common/src/chain/upgrades/netupgrade.rs

```diff
@@ -42,6 +42,19 @@ impl NetUpgrades<UpgradeVersion> {
         )])
     }
 
+    pub fn deliberate_ignore_consensus_twice() -> Self {
+        Self(vec![
```

Comment:
As far as I can see, this is only used in tests. Should it be guarded by `#[cfg(test)]`?

Reply:
Done